### ISSUE TITLE
Feature: sort room filters like on the official website 

### ISSUE THREAD

**Issue by jtanguy** [GDG-Nantes/DevfestNantesMobile#88]
It seems that the room filters are sorted alphabetically, instead of the order presented on the website (it seems that they are ordered by capacity, but I'm not sure). 
It I'll be easier to use the room filters (the big room first instead of third) 

![Screenshot_20221017-142300.jpg](https://user-images.githubusercontent.com/551107/196176236-8758b625-4370-428a-887f-e79158dd8921.jpg)

**Comment by RobinCaroff** [comment]
@jtanguy Thanks for submitting the issue! Good thinking, it would make more sens indeed.

We cannot automatically deduce the importance of the rooms from the data provided by the DevFest Nantes's repo(https://github.com/GDG-Nantes/Devfest2022/tree/master/data) and we would need this info to setup sorting on our backend side.

I'll keep this issue opened as I'm sure we can do this for next year.

**Comment by jtanguy** [comment]
We could keep the order in the rooms. Inside `schedule.json`, there is an array of rooms, which as a sort

**Comment by jtanguy** [comment]
Did we loose the sort when casting into a `Set` ?

**Comment by RobinCaroff** [comment]
Good idea! Yes indeed we could use this order; however I'll need to update the sorting algorithm on the app's side as it groups sessions by start times but the does not sort the sessions by rooms. I fear that the update won't make it to production before the start of the event (review process is currently awfully long...). I'll let you know how it goes!

**Comment by RobinCaroff** [comment]
Done for Android in #149
